gufe/protocols/protocolunit.py

```diff
@@ -323,16 +295,19 @@ def execute(self, *,
                           scratch=scratch_)
 
         try:
-            outputs = self._execute(context, **inputs)
+            outputs = self._execute(context, **dependencies)
             result = ProtocolUnitResult(
-                name=self.name, source_key=self.key, inputs=inputs, outputs=outputs
+                name=self.name,
+                source_key=self.key,
+                dependencies=dependencies,
+                outputs=outputs
```

Comment:
I think we'll still need a separate `inputs` kwarg here? Otherwise we're dropping some of the `ProtocolUnit` state on the floor when we create its `ProtocolUnitResult`.